CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 Magento Functional Testing Framework Changelog
 ================================================
 
+2.4.0
+-----
+### Enhancements
+* Maintainability
+    * Added new `mftf static-checks` command to run existing static checks against the attached test codebase
```

Comment:
Why called it "existing" static checks? Isn't it the "new" static checks? We do have some existing static checks and they are different from this one. Does it make sense?

Reply:
Good point, I'll change the wording here